public/js/selfoss-events-entries.js

```diff
@@ -169,6 +169,13 @@ selfoss.events.entries = function() {
         });
     }
 
+    // stream pagination buttons
+    $('#stream-pagination button').unbind('click').click(function() {
+        var pageNumber = parseInt($(this).html());
```

Comment:
In this case, we control the values parsed but it would still be prudent to explicitly specify the radix.

```suggestion
        var pageNumber = parseInt($(this).html(), 10);
```

I am surprised eslint does not shout about this, [the rule](https://eslint.org/docs/rules/radix) should be on by default.